README.md

```diff
@@ -1,2 +1,5 @@
-# class-01-lab
-Class 01 Lab
+# Server Deployment Practice
```

Comment:
format you links for example: `[Pull Request](LINK TO PULL REQUEST)` that way it shows up like this: [Pull Request](www.xyz.com), Label the other links as Github Actions, and Deployed App.

Then add details on how you application works and how to run your application. If you want you can think of this section as notes, or a way to help remind you what this application is and how to make it, so if you want to re-visit this lab in 4-6 months you have a better idea what it about. Or write it as someone is going to use you app.